src/modules/imageresizer/tests/Properties/SettingsTests.cs

```diff
@@ -30,9 +30,10 @@ public void AllSizesPropagatesSizesCollectionEvents()
         {
             var settings = new Settings
             {
-                Sizes = new ObservableCollection<ResizeSize>(),
                 CustomSize = new CustomSize(),
             };
+
+            settings.Sizes.Clear();
```

Comment:
Why do we need these `Clear` calls?

Reply:
Since before the observable collection was reset by setting to new obj. Fxcop said that was a No no